python/ccf/proposal_generator.py

```diff
@@ -225,83 +227,27 @@ def cli_proposal(func):
 
 
 @cli_proposal
-def new_member(
+def set_member(
     member_cert_path: str,
     member_enc_pubk_path: str = None,
     member_data: Any = None,
     **kwargs,
 ):
-    LOG.debug("Generating new_member proposal")
+    LOG.debug("Generating set_member proposal")
```

Comment:
```suggestion
```

Since `set_member` is now implemented via `build_proposal` like everything else, it shouldn't have custom logging.